### PR TITLE
Windows support

### PR DESCRIPTION
**Pull request by kswenson**
- add polyfills for IE11 support
- add text adjustments for all Windows browsers

All Windows browsers tested (Chrome, Firefox, IE11) showed the same text offsets, suggesting that it may be a font-metrics issue rather than a browser issue. We "fix" it for the time being by user agent-sniffing for Windows and adding the `os-windows` class to the `<body>` tag if appropriate. Then CSS is used on a case-by-case basis to adjust the text where appropriate.

Note: after review, let me merge so I can rebase appropriately.

### CODE REVIEW

File: src/code/containers/modal-message-container.js [hunk 1]
```diff
@@ -2,7 +2,8 @@ import { connect } from 'react-redux';
 import ModalAlert from '../components/modal-alert';
 import * as actions from '../actions';
 import { resetGametes } from '../modules/gametes';
-Object.assign(actions, { resetGametes });
+// merge with the other actions
+actions.resetGametes = resetGametes;
```
Comment (ekosmin) [inline src/code/containers/modal-message-container.js]: Is this change necessary, now that `Object.assign` works on all browsers?

File: src/stylus/fablevision/parent-drake.styl [hunk 2]
```diff
@@ -33,3 +33,8 @@
     margin-top: 10px
   .organism
     margin-top: -8px
+
+.os-windows
+  .geniblocks.parentDrake
+    .parentTitle
+      margin-top: 1rem
```
Comment (ekosmin) [inline src/stylus/fablevision/parent-drake.styl]: Were you able to find any consistent way to determine how much the Windows offset needs to be? Checking for every text component seems tedious

File: src/code/containers/modal-message-container.js (hunk 1 above)
Comment (kswenson) [inline src/code/containers/modal-message-container.js]: Still not in IE11.

File: src/stylus/fablevision/parent-drake.styl (hunk 2 above)
Comment (kswenson) [inline src/stylus/fablevision/parent-drake.styl]: Not yet, but I haven't looked very hard. I think manually tweaking the handful we care about for the classroom test makes sense, and then we can take the time to look for a more general solution after the classroom test.

File: src/code/containers/modal-message-container.js (hunk 1 above)
Comment (ekosmin) [inline src/code/containers/modal-message-container.js]: My impression was that the purpose of the polyfill was to make it work in IE11, but I must misunderstand. We can catch up about it offline

File: src/stylus/fablevision/parent-drake.styl (hunk 2 above)
Comment (ekosmin) [inline src/stylus/fablevision/parent-drake.styl]: Makes sense!

File: src/code/containers/modal-message-container.js (hunk 1 above)
Comment (kswenson) [inline src/code/containers/modal-message-container.js]: Ah, the reason I eliminated the use of Object.assign in this particular case was because I didn't like the idea of having this module be responsible for loading the polyfill rather than having it handled centrally. The only reason that was necessary was because this module was being loaded by the browser before the modules that loaded the polyfill. But rather than figure that out, it was easier to eliminate the use of Object.assign() in this case.
